uberenv.py

```diff
@@ -770,11 +776,7 @@ def find_spack_pkg_path(self, pkg_name, spec = ""):
 
     # Extract the first line of the full spec
     def read_spack_full_spec(self,pkg_name,spec):
-        debug = ""
-        if self.opts["spack_debug"]:
-            debug = "--debug --stacktrace "
-
-        res, out = sexe("{0} {1}spec {2}".format(self.spack_exe(),debug,self.pkg_name_with_spec), ret_output=True)
+        res, out = sexe("{0} spec {2}".format(self.spack_exe(), self.pkg_name_with_spec), ret_output=True)
```

Comment:
```suggestion
        res, out = sexe("{0} spec {1}".format(self.spack_exe(), self.pkg_name_with_spec), ret_output=True)
```

Reply:
Thanks for catching this. This function is never being called anywhere by the way-- not sure if we need to keep it or not.

Reply:
If it isn't being called you can just remove it.